python/cuml/neighbors/nearest_neighbors.pyx

```diff
@@ -453,7 +453,7 @@ class NearestNeighbors(Base,
     def get_param_names(self):
         return super().get_param_names() + \
             ["n_neighbors", "algorithm", "metric",
-                "p", "metric_params", "algo_params"]
+                "p", "metric_params", "algo_params", "n_jobs"]
```

Comment:
Can we add a little pytest case to validate this? It will help in the future to know that this is working as expected.

Reply:
Good idea, @cjnolet !  Thank you for the comment

Reply:
Thank you, @cjnolet;  let me know what you think!